test/functional/test_framework/script.py

```diff
@@ -747,7 +747,7 @@ def SegwitV0SignatureMsg(script, txTo, inIdx, hashtype, amount):
     ss += struct.pack("<q", amount)
     ss += struct.pack("<I", txTo.vin[inIdx].nSequence)
     ss += ser_uint256(hashOutputs)
-    ss += struct.pack("<i", txTo.nLockTime)
+    ss += txTo.nLockTime.to_bytes(4, "little")
```

Comment:
Could we keep the format?
```suggestion
    struct.pack("<I", txTo.nLockTime)
```

In the instructions you've mentioned it's reproducible - can you add a test that fails before this change, but passes after?

Reply:
> can you add a test that fails before this change, but passes after?

Wouldn't that mean that we added a test that tests the test-framework?